scio-avro/src/main/scala/com/spotify/scio/avro/types/MacroUtil.scala

```diff
@@ -49,7 +49,7 @@ private[types] object MacroUtil {
 
   val ScioAvro = "_root_.com.spotify.scio.avro"
   val ApacheAvro = "_root_.org.apache.avro"
-  val ScioAvroType = s"$ScioAvro.types.AvroType"
+  val ScioAvroType: String = s"$ScioAvro.types.AvroType"
```

Comment:
Does `ScioAvroType` require a type annotation but not `ApacheAvro`?

Reply:
couldn't find yet a solution for cases where string interpolation is used.

Reply:
I can leave with that for now though.